README.md

```diff
@@ -7,6 +7,9 @@ This project allows you to get started a nextcloud server and develop your first
 This project is very **modular** where you can add [features](#different-feature-you-can-use).
 So, you can use this project for your development environment only.
 
+If you don't know how to install Docker, please, read this documentation : https://cloud.nextcloud.com/s/iyNGp8ryWxc7Efa .
+Be careful : Read the "README.md" first.
```

Comment:
Might be a bit confusing that it is about the repo readme ;)

Reply:
Yes, I know, I thought so too ^^

Reply:
What do you think with this change : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/61db98eeee42d5afab9ad33d6744e2add679948f ?